misc/plugutils.c

```diff
@@ -424,7 +425,15 @@ proto_post_wrapped (const char *oid, struct script_infos *desc, int port,
                             msg_type_to_str (msg_type), ip_str, hostname ?: " ",
                             port_s, proto, oid, action_str->str, uri ?: "");
   /* Convert to UTF-8 before sending to Manager. */
```

Comment:
I suppose this comment isn't valid anymore because openvas doesn't send anything to the manager anymore?

Reply:
At the end the result still end up in gvmd.
So instead of `openvas-scanner->redis->gvmd` its now `openvas-scanner->redis->ospd-openvas->gvmd`.
I think I would leave the comment as is, as hints at problems which might occur on manager side when the strings are not converted. But I don't deeply understand the issues with encoding problems. I could e.g. not get the `g_convert()` to fail on purpose.